internal/agent/install.go

```diff
@@ -82,6 +84,10 @@ func ManualInstall(c, device string, reboot, poweroff, strictValidations bool) e
 }
 
 func Install(dir ...string) error {
+	var cc *config.Config
+	var err error
+
+	bus.Manager.Initialize()
```

Comment:
this is already done on Main during command init, but I seen that reset and upgrade have it also in their functions, in case they are called outside of the main function somehow?

Seems like adding it doesnt hurt and could prevent issues.